pkg/reconciler/v1alpha1/revision/resources/env_var.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"github.com/knative/serving/pkg/apis/serving"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	knativeRevisionEnvVariableKey      = "KNATIVE_REVISION_NAME"
```

Comment:
I am not sure of the value of using `_NAME` as a suffix. I would prefer to remove it in favor of `KNATIVE_REVISION`, or simply `K_REVISION`

Reply:
+1 -- good catch, @steren 

I don't have a strong preference for `K_` vs `KNATIVE_`. Existing systems seem to mostly use short names (`AWS`, `OW`, `CF`, etc), which would argue for `K_` as a prefix.

Reply:
will change them to `K_` and remove `_NAME`